docs/rm/snax_cluster.md

```diff
@@ -0,0 +1,198 @@
+# SNAX Cluster Documentation
+
+This section discusses the modifications to the Snitch cluster to makthe acceleration extensions as easy as possible. The figure below shows a simplified figure of the SNAX architecture.
+
+![SNAX Cluster](https://drive.google.com/uc?id=1PqMhcxbqWnJdn-EawCfGetc0CbqfFCK_)
```

Comment:
Maybe mention that the number of TCDM banks is parametrizable to other values than 32?